src/main/java/com/spartronics4915/frc2019/subsystems/Drive.java

```diff
@@ -478,7 +479,10 @@ private void updateVelocity()
         }
 
     }
-
+    public Rotation2d getPitch()
+    {
+        return mPeriodicIO.gyro_pitch;
+    }
```

Comment:
Whitespace nitpick: put a line break between the end of this method and the start of the next.